feat(onp): manage quotas page (#3333)
scaleway/docs-content: identity-and-access-management/organizations-and-projects/how-to/manage-quotas.mdx

---
meta:
  title: How to manage Organization quotas
  description: This page shows you how to view and manage Scaleway Organization quotas.
content:
  h1: How to manage Organization Quotas

>   paragraph: his page shows you how to view and manage Scaleway Organization quotas.

Reviewer: ```suggestion
  paragraph: This page shows you how to view and manage Scaleway Organization quotas.
```

tags: account-quotas quotas security-rule security rule
dates:
  validation: 2024-06-17

>   posted: 2021-02-10

Reviewer: ```suggestion
  posted: 2024-06-17
```

categories:
  - console
---

<Macro id="requirements" />

- A Scaleway account logged into the [console](https://console.scaleway.com)
- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization

## How to view Organization quotas

1. Click the **Quotas** tab from the [Organization Dashboard](https://console.scaleway.com/organization). A list of all quotas displays.
2. Click the name of the resource you want to view the quotas for.
3. Click <Icon name="more" /> > **More info** next to the name of your resource of choice. A pop-up appears.

  In the pop-up you can find more information about the quota, such as **Name**, **Technical name**, **Product**, **Max quota per Availability Zone**, **Current usage**, and **Description**.

You can also use the menu at the top of the page to search and filter your quotas.

<Lightbox src="steps-to-filter-quotas.webp" alt="" />

As shown in the image above, you can:

> 1. Search by the precise name of the resource.

Reviewer: ```suggestion
1. Search quotas by specifying the name of the resource.
```

2. Specify the product you want to see the quotas for.

> 3. Specify the location of your resources. This includes Regions, Availability Zones, or all locations.

Reviewer: ```suggestion
3. Specify the location of your resources. This includes regions, Availability Zones, or all locations.
```

4. Check the box to see used quotas only.
5. Reset your filters.
6. Click **Filter** to apply your choices.

## How to increase quotas

Quotas are assigned by default according to each account level.

There are two ways you can increase your account level:
    - By [validating your payment method](/console/billing/how-to/add-payment-method/#how-to-add-a-credit-card)
    - By [verifying your identity](/console/account/how-to/verify-identity/)

For a detailed list of available quotas and the account levels necessary to increase quota capacity, as well as availability per location, refer to the [Understanding Organization Quotas](/identity-and-access-management/organizations-and-projects/additional-content/organization-quotas/) documentation page.

<Message type="note">

>   If you have already validated your payment method and your identity and want to increase your quota beyond the values shown in the console or on the [Understanding Organization Quotas](/identity-and-access-management/organizations-and-projects/additional-content/organization-quotas/) documentation page, you must [contact our support team](https://console.scaleway.com/support/create).

Reviewer: ```suggestion
  If you have already validated your payment method and identity and want to increase your quota beyond the values shown in the console or on the [Understanding Organization Quotas](/identity-and-access-management/organizations-and-projects/additional-content/organization-quotas/) documentation page, you must [contact our support team](https://console.scaleway.com/support/create).
```
